genrl/environments/vectorised_envs.py

```diff
@@ -46,6 +46,8 @@ class VecEnv(ABC):
     def __init__(self, env, n_envs=2):
         self.envs = create_envs(env, n_envs)
         self._n_envs = len(self.envs)
+        self.observation_space = self.env.observation_space
+        self.action_space = self.env.action_space
```

Comment:
why do we need this? I mean I already have it in worker method in get_spaces.

Reply:
This is supposed to work for both regular envs and vectorised envs. So if we use this as a wrapper, we need to be directly be able to call `env.observation_space` like you would call normally. My aim was not to affect normal usage.